src/sentry/static/sentry/app/components/organizationIssueList.jsx

```diff
@@ -50,7 +50,7 @@ const OrganizationIssueList = React.createClass({
           </div>
         </div>
         <h3>{this.props.title}</h3>
-        <div className="alert alert-block alert-info">Psst! This feature is still a work-in-progress. Thanks for being an early adopter!</div>
+        <div className="alert alert-block alert-info">{'Psst! This feature is still a work-in-progress. Thanks for being an early adopter!'}</div>
```

Comment:
My thinking was that early adopter messages are probably not going out in multiple languages, <shrug>

cc @dcramer